web/src/app/util/SetFavoriteButton.tsx

```diff
@@ -42,4 +44,22 @@ export function SetFavoriteButton({
       </IconButton>
     </form>
   )
+
+  // todo: add other types as needed
```

Comment:
Did we create a story for this or will we just spot in future (by searching todos)?

Reply:
I didn't, I left it in as a todo when it was an API change, but now that we are using `typename` and can hardcode the strings as needed, I think we can delete this todo